nemo_skills/pipeline/summarize_results.py

```diff
@@ -150,6 +150,15 @@ def summarize_results(
                         aggregation_mode="best",
                         max_samples=max_samples,
                     )
+                    try:
+                        results[benchmark][f'rw@{len(sampling_outputs)}'] = compute_metrics(
+                            input_files=sampling_outputs,
+                            metrics_calculator=metrics_calculator,
+                            aggregation_mode="reward",
+                            max_samples=max_samples,
+                        )
+                    except:
```

Comment:
Using blank except is generally not a good idea as it might be hard to notice real errors. How about we take this moment to refactor this script to address https://github.com/Kipok/NeMo-Skills/issues/184 which will make adding new aggregations much easier in the future. Instead of taking the aggregation_mode as a parameter, compute metrics should instead return a dictionary with all available aggregations and we would just print all of them here. Let me know if it's not clear and I can try to explain better or we can sync offline

Reply:
I revised the code as above, please take a look if it aligns with the goal of https://github.com/Kipok/NeMo-Skills/issues/184, then I can keep revising